modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -950,6 +994,8 @@ public void stop() {
 
         busyLock.block();
 
+        metaStorageMgr.unregisterWatch(zonesWatchListener);
```

Comment:
To be honest we don't really need this. Unregister will just remove some filters from the volatile meta storage state, that is useless in case of node stop. I mean that given unregister isn't about distributed watch de-registration. That also means that you don't need `private volatile WatchListener zonesWatchListener; ` as an explicit variable.